app/models/container_label_tag_mapping.rb

```diff
@@ -59,41 +66,33 @@ def self.tags_for_name_type_value(name, type, value)
       specific_value
     else
       any_value.map do |category_tag|
-        create_specific_value_mapping(name, type, value, category_tag).tag
+        specific_value_tag(name, value, category_tag)
       end
     end
   end
   private_class_method :tags_for_name_type_value
 
   # If this is an open ended any-value mapping, finds or creates a
   # specific-value mapping to a specific tag.
-  def self.create_specific_value_mapping(name, type, value, category_tag)
-    new_tag = create_tag(name, value, category_tag)
-    new_mapping = create!(:labeled_resource_type => type, :label_name => name, :label_value => value,
-                          :tag => new_tag)
-    load_mapping_into_hash(new_mapping)
-    new_mapping
-  end
-  private_class_method :create_specific_value_mapping
-
-  def self.create_tag(name, value, category_tag)
+  def self.specific_value_tag(name, value, category_tag)
     category = category_tag.classification
-    unless category
-      category = Classification.create_category!(:description => "Kubernetes label '#{name}'",
-                                                 :read_only   => true,
-                                                 :tag         => category_tag)
-    end
 
+    # Note: the names chosen here should remain stable,
+    # or we won't be able to find previously created tags.
```

Comment:
@cben I thought we said we should remove the `:empty:` dummy value. I understand that it may not be for this PR but adding this comment seems useless since we're going to remove it.